pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/fraser/barnepensjon/BarnepensjonFellesFraser.kt

```diff
@@ -186,51 +192,122 @@ object BarnepensjonFellesFraser {
             showIf(brukerUnder18Aar) {
                 paragraph {
                     text(
-                        Bokmal to "Barnepensjon er skattepliktig, men ikke trekkpliktig. Du kan lese mer om skattetrekk i vedlegget “Informasjon til deg som handler på vegne av barnet”.",
-                        Nynorsk to "Barnepensjon er skattepliktig, men ikkje trekkpliktig. Du kan lese meir om skattetrekk i vedlegget “Informasjon til deg som handlar på vegne av barnet”.",
-                        English to "Child pension is taxable, but not subject to withholding tax. You can read more about tax deductions in the attachment “Information for those acting on behalf of the child”.",
-                        )
+                        Bokmal to "Barnepensjon er skattepliktig, men ikke trekkpliktig.",
+                        Nynorsk to "Barnepensjon er skattepliktig, men ikkje trekkpliktig.",
+                        English to "Child pension is taxable, but not subject to withholding tax.",
+                    )
                 }
             } orShow {
                 paragraph {
                     text(
-                        Bokmal to "Barnepensjon er skattepliktig, men ikke trekkpliktig. Du kan lese mer om skattetrekk i vedlegget “Informasjon til deg som mottar barnepensjon”.",
-                        Nynorsk to "Barnepensjon er skattepliktig, men ikkje trekkpliktig. Du kan lese meir om skattetrekk i vedlegget “Informasjon til deg som får barnepensjon”.",
-                        English to "Child pension is taxable, but not subject to withholding tax. You can read more about tax deductions in the attachment “Information to recipients of children’s pensions”.",
+                        Bokmal to "Barnepensjon er skattepliktig, men ikke trekkpliktig.",
```

Comment:
Trenger man denne lenger når de to sidene av if'en er like? Tidligere var det jo forskjellig vedlegg avhengig av alder.

Reply:
Nei, det kan forenkles 👍